tools/configen/conf/configen.yaml

```diff
@@ -15,7 +15,10 @@ configen:
   # list of modules to generate configs for
   modules:
     - name: configen.samples.my_module
-      _convert_: 'all'
+      # Optional dict defining hydra flags for all classes in this module
+      default_flags:
+        _convert_ : 'all'
+        _recursive_ : True
```

Comment:
Let's not change the sample config output. Can you document it in a comment?

Reply:
yes